requirements3.txt

```diff
@@ -0,0 +1,7 @@
+testresources==2.0.1
```

Comment:
Based on @rshaull's [comment](https://github.com/nuodb/nuodb-collector/issues/23#issuecomment-1353723140), this should not be necessary.

Even if it is due to the cache issue (which he suggested a workaround for), we do not need two requirements files. You can use environment marker to say what the version range should be depending on the Python version. See below:
https://pip.pypa.io/en/stable/reference/requirement-specifiers/

I think the single requirements.txt should have something like:
```
requests >= 2.24.0, < 2.28 ; python_version < '3.0'
requests >= 2.24.0 ; python_version >= '3.0'
```

Reply:
okay,   I did not know how to do this with one requirements file.   I can rewrite.  makes instructions for deploying easier.